server.js

```diff
@@ -1,5 +1,49 @@
-import app from './app.js';
+import cors from 'cors';
+import express, { json } from 'express';
+import mongoose from 'mongoose';
+import 'dotenv/config';
+import router from './api/index.js';
 
-app.listen(3000, () => {
-  console.log('Server running. Use our API on port: 3000');
+const app = express();
+
+// parse application/json
```

Comment:
Te komentarze są bez sensu. Po co tam komentarz cors jak po samej nazwie funkcji wiesz od czego to jest. Ogólnie zasada jest taka że piszemy kom jak coś jest nie jasne, a jeśli coś jest nie jasne to znaczy że zwaliłeś nazewnictwo i trzeba to napisać jeszcze raz.

Reply:
Fixed.